examples/validate_relative_binding.py

```diff
@@ -45,6 +45,26 @@
 from md import builders, minimizer
 from rdkit.Chem import rdFMCS
 
+
+def cache_wrapper(cache_path: str, f):
```

Comment:
Should we add an optional argument `overwrite=False`? (Perhaps a brief comment about when and how we should invalidate the cache would be helpful too!)